iphone/Maps/UI/Search/MWMSearchManager.mm

```diff
@@ -471,13 +471,22 @@ - (void)viewHidden:(BOOL)hidden {
     [parentView addSubview:contentView];
     [parentView addSubview:actionBarView];
     [self layoutTopViews];
+    // Set Search controller default hidden state for iPad before it will be shown.
+    if (IPAD)
+      self.searchViewContainerLeadingConstraint.constant = -kWidthForiPad;
+      [parentView.superview layoutIfNeeded];
```

Comment:
There is no bug here? 2 expressions should be under if block?
If no - fix indent.